File: Backend/htbp_parent/service/service_order/src/main/java/com/seclib/htbp/order/ServiceOrderApplication.java
package com.seclib.htbp.order;

import com.seclib.htbp.order.mapper.OrderInfoMapper;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackages = {"com.seclib.htbp"})
@EnableDiscoveryClient
@EnableFeignClients(basePackages = {"com.seclib.htbp"})
public class ServiceOrderApplication {

    public static void main(String[] args) {
        SpringApplication.run(ServiceOrderApplication.class, args);
    }
}

File: Backend/htbp_parent/service/service_order/src/main/java/com/seclib/htbp/order/ServiceOrderApplication.kt
package com.seclib.htbp.order

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.cloud.openfeign.EnableFeignClients
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@ComponentScan(basePackages = ["com.seclib.htbp"])
@EnableDiscoveryClient
@EnableFeignClients(basePackages = ["com.seclib.htbp"])
open class ServiceOrderApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ServiceOrderApplication::class.java, *args)
        }
    }
}